docs/deployment/rds-s3/README.md

```diff
@@ -2,13 +2,42 @@
 
 ## Overview
 
-This Kustomize Manifest can be used to deploy Kubeflow Pipelines and Katib with RDS and S3.
+This Kustomize Manifest can be used to deploy Kubeflow Pipelines (KFP) and Katib with RDS and S3.
 
-Follow the [install](#install) steps below to configure and deploy the Kustomize manifest.
+### RDS
+
+[Amazon Relational Database Service (RDS)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) is a managed relational database service that facilitates several database management tasks such as database scaling, database backups, database software patching, OS patching, and more.
+
+In the [default kubeflow installation](../../../../example/kustomization.yaml), the [KFP](../../../../apps/katib/upstream/components/mysql/mysql.yaml) and [Katib](../../../../apps/pipeline/upstream/third-party/mysql/base/mysql-deployment.yaml) components both use their own MySQL pod to persist KFP data (such as experiments, pipelines, jobs, etc.) and Katib experiment observation logs, respectively. 
+
+As compared to using the MySQL setup in the default installation, using RDS provides the following advantages:
+- Easier to configure scalability and availability: RDS provides high availability and failover support for DB instances using Multi-AZ deployments with a single standby DB instance, increasing the availability of KFP and Katib services during unexpected network events.
+- Persisted KFP and Katib data can be reused across Kubeflow installations with the same Kubeflow version: Using RDS decouples the KFP and Katib datastores from the Kubeflow deployment, allowing the same RDS instance to be used across multiple Kubeflow installations with the same Kubeflow version.
```

Comment:
multiple kubeflow installations writing to same tables and databases?

Reply:
Shall we call out the EBS PVC disadvantages from default Kubeflow? - AZ bound and no automatic online resizing.

Reply:
> multiple kubeflow installations writing to same tables and databases?

reworded this

> Shall we call out the EBS PVC disadvantages from default Kubeflow? - AZ bound and no automatic online resizing.

done